metric-anomaly-task-manager/src/main/java/org/hypertrace/alert/engine/metric/anomaly/task/manager/job/AlertTaskConverter.java

```diff
@@ -41,9 +50,14 @@ public AlertTaskConverter(Config jobConfig) {
     this.jobConfig = jobConfig;
   }
 
-  public AlertTask.Builder toAlertTaskBuilder(Document document) throws IOException {
+  public Optional<AlertTask.Builder> toAlertTaskBuilder(Document document) throws IOException {
     JsonNode rule = OBJECT_MAPPER.readTree(document.toJson());
 
+    if (!validateRule(rule)) {
+      LOGGER.info("Invalid AlertTask : {}", rule);
```

Comment:
```suggestion
      LOGGER.info("Invalid Alerting rule: {}", rule);
```

Reply:
Is it possible to add what's wrong with the message? Can we move the logging to `validateRule` so we can tell what's specifically wrong with this rule? And so, it will be easy to correct.
Or we can add more info here. 
LOGGER.info(`Aggregation interval should be 15/30/60s. Baseline, and rule duration should be in minutes. Skipping Invalid alerting rule from evaluation {}", rule)`

Reply:
Here, we can log as `LOGGER.info("Skipping Invalid alerting rule from evaluation")`

Reply:
@kotharironak Do you want different log message for different validation failure (like "failure : baseline duartion not in minutes") or one common message (like the one you posted above) ?

Reply:
we can go with this - `LOGGER.info("The aggregation interval should be 15/30/60s. Baseline and rule duration should be in minutes. Skipping Invalid alerting rule from evaluation {} as not meeting the criteria", rule)`